docs/community_snippets.md

```diff
@@ -313,9 +313,9 @@ An example looks like this (NOTE: all lowercase link):
 
 which will show up in the ToC like this:
 
-| Table of Contents |
-|:------------------|
-| [PSCustomObject](#pscustomobject): _A simple PSCustomObject_ |
+| Snippet Name | Description |
+|--------- | ---------|
```

Comment:
There's also one more change to this a little bit higher up on line 305. Can you also fix that one?

Reply:
I missed that. I've fixed it.